### ISSUE TITLE
Missing heading levels

### ISSUE THREAD

**Issue by cvinczefullsail** [keithoneill/keithoneill.github.io#12]
Your pages are missing h1 and h2 headings. The outline of your pages would confuse screenreaders and search engines.
